lib/coinbase/signer.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'client'
+
+module Coinbase
+  # A representation of a Signer. Signers are assigned to sign transactions for a Wallet.
+  class Signer
```

Comment:
we should call this ServerSigner IMO since thats what our docs call it https://docs.cdp.coinbase.com/wallets/docs/serversigners